package.json

```diff
@@ -43,7 +43,7 @@
     "debug": "^4.3.1",
     "filing-cabinet": "^3.0.1",
     "precinct": "^9.0.0",
-    "typescript": "^4.0.0"
+    "typescript": "^4.0.0 || ^5.0.0"
```

Comment:
Does this make sense though? This isn't peerDependencies, it's dependencies. I suggest that you just use v5 here and the next version is major bump.

Reply:
moving it to ^5.0.0 then

Reply:
I initially went for `^4||^5` to avoid implying any change on your clients. As TS does not follow semver at all, actually even `>=4` would have been correct 😭